### ISSUE TITLE
Whitespace between expressions is lost

### ISSUE THREAD

**Issue by kiejo** [ryansolid/babel-plugin-jsx-dom-expressions#8]
It looks like something like this:
```jsx
function Component({one, two}) {
  return <span>{one} {two}</span>
}
```
gets compiled to this:
```javascript
const _tmpl$ = document.createElement("template");
_tmpl$.innerHTML = "<span><!--2--><!--3--></span>";
```

As you can see the space between the two expressions is lost, which is probably not the expected behavior.

**Comment by ryansolid** [comment]
Thanks for reporting. I must of somehow never hit a situation where there wasn't atleast one other character in between. I had definitely thought JSX Text parsing AST generated differently and it was getting skipped based on completely different code unrelated to the whiteSpace trim method I use. So this issue gave me the chance to really dig into this and figure out this pretty important bug.

Text Interpolation is arguably still an area that might need a bit more consideration. I've followed suite with HTML compressing extra whitespace but I not 100% sure it is the expected behavior.

Anyway, fixed in v0.7.4. Hope this works for you.

**Comment by kiejo** [comment]
Looks good, thanks for the fix!
